targets/TARGET_STM/TARGET_STM32WL/README.md

```diff
@@ -31,11 +31,22 @@ Only M4 core is currently used by mbed-os applications.
 - Total FLASH is 256KB
 - RAM: 64 KB
 
+**[tip] mbed-os export feature**
 
-## CUSTOM boards
+`mbed export -m NUCLEO_WL55JC -i uvision6`
```

Comment:
This should not be part of this PR neither the commit (not related to the PR ,is it?)

Reply:
It's about WL... :-)
I could make 2 commits at least !

Reply:
true lets separate these. Should we however document deprecated features (at least a note there ?)

Reply:
@0xc0170 done